ts-client/src/dlmm/index.ts

```diff
@@ -5964,4 +5990,241 @@ export class DLMM {
+      binLiquidityDistribution,
+    };
+  }
+
+  private async preparePositionAndBinArrayForSeeding(
+    lowerBinId: BN,
+    upperBinId: BN,
+    base: PublicKey,
+    position: PublicKey,
+    operator: PublicKey,
+    owner: PublicKey,
+    feeOwner: PublicKey,
+    appendedInitBinArrayIx: Set<PublicKey>,
+    lockReleasePoint: BN
+  ) {
+    const lowerBinArrayIndex = binIdToBinArrayIndex(lowerBinId);
+    const upperBinArrayIndex = binIdToBinArrayIndex(upperBinId);
+
+    const [lowerBinArray] = deriveBinArray(
+      this.pubkey,
+      lowerBinArrayIndex,
+      this.program.programId
+    );
+
+    const [upperBinArray] = deriveBinArray(
+      this.pubkey,
+      upperBinArrayIndex,
+      this.program.programId
+    );
+
+    const accounts =
+      await this.program.provider.connection.getMultipleAccountsInfo([
+        lowerBinArray,
+        upperBinArray,
+        position,
+      ]);
+
+    const instructions: TransactionInstruction[] = [];
+    const lowerBinArrayAccount = accounts[0];
+
+    if (
+      !lowerBinArrayAccount &&
+      !appendedInitBinArrayIx.has(lowerBinArray)
+    ) {
+      instructions.push(
+        await this.program.methods
+          .initializeBinArray(lowerBinArrayIndex)
+          .accounts({
+            lbPair: this.pubkey,
+            binArray: lowerBinArray,
+            funder: operator,
+          })
+          .instruction()
+      );
+    }
+
+    const upperBinArrayAccount = accounts[1];
+    if (
+      !upperBinArrayAccount &&
+      !appendedInitBinArrayIx.has(upperBinArray)
+    ) {
+      instructions.push(
+        await this.program.methods
+          .initializeBinArray(upperBinArrayIndex)
+          .accounts({
+            lbPair: this.pubkey,
+            binArray: upperBinArray,
+            funder: operator,
+          })
+          .instruction()
+      );
+    }
+
+    const positionAccount = accounts[2];
+    if (!positionAccount) {
+      instructions.push(
+        await this.program.methods
+          .initializePositionByOperator(
+            lowerBinId.toNumber(),
+            MAX_BIN_PER_POSITION.toNumber(),
+            feeOwner,
+            lockReleasePoint
+          )
+          .accounts({
+            lbPair: this.pubkey,
+            position,
+            base,
+            owner,
+            ownerTokenX: getAssociatedTokenAddressSync(
+              this.tokenX.publicKey,
+              owner,
+              true
+            ),
+            operator,
+            operatorTokenX: getAssociatedTokenAddressSync(
+              this.tokenX.publicKey,
+              operator
+            ),
+            payer: operator,
+          })
+          .instruction()
+      );
+    }
+
+    // Initialize bin arrays and initialize position account in 1 tx
+    if (instructions.length > 1) {
+      instructions.unshift(
+        await getEstimatedComputeUnitIxWithBuffer(
+          this.program.provider.connection,
+          instructions,
+          owner
```

Comment:
the param is `feePayer` so it should be`operator`?

Reply:
Fixed